client/src/Home.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LayerHost, Spinner, Stack, ThemeProvider } from '@fluentui/react';
+import { backgroundStyles, fullSizeStyles } from './styles/Common.styles';
+// import { Header } from './Header';
+import { useEffect, useState } from 'react';
+import { fetchConfig } from './utils/FetchConfig';
+import { GenericError } from './components/GenericError';
+import { AppConfigModel } from './models/ConfigModel';
+import { HomeComponent } from './components/home/Home';
+import { Header } from './Header';
+import './styles/Common.css';
+
+const PARENT_ID = 'HomeSection';
+
+export const Home = (): JSX.Element => {
+  const [config, setConfig] = useState<AppConfigModel | undefined>(undefined);
+  const [error, setError] = useState<any | undefined>(undefined);
+
+  useEffect(() => {
+    const fetchData = async (): Promise<void> => {
```

Comment:
I am not really sure if this is necessary. For me, this page is more static website and it doesn't look like we should fetch any configs.

Please share any information if I am missing something.

Reply:
Actually I found out that we need `config` because of the theme.